bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/handler/RenaultHandler.java

```diff
@@ -104,13 +105,70 @@ public void initialize() {
         }
         updateStatus(ThingStatus.UNKNOWN);
 
+        updateState(CHANNEL_HVAC_TARGET_TEMPERATURE,
+                new QuantityType<Temperature>(car.getHvacTargetTemperature().doubleValue(), SIUnits.CELSIUS));
+
         // Background initialization:
         ScheduledFuture<?> job = pollingJob;
         if (job == null || job.isCancelled()) {
             pollingJob = scheduler.scheduleWithFixedDelay(this::getStatus, 0, config.refreshInterval, TimeUnit.MINUTES);
         }
     }
 
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        switch (channelUID.getId()) {
+            case RenaultBindingConstants.CHANNEL_HVAC_TARGET_TEMPERATURE:
+                if (command instanceof RefreshType && !car.isDisableHvac()) {
+                    updateState(CHANNEL_HVAC_TARGET_TEMPERATURE, new QuantityType<Temperature>(
+                            car.getHvacTargetTemperature().doubleValue(), SIUnits.CELSIUS));
+                } else if (command instanceof QuantityType) {
+                    car.setHvacTargetTemperature(((QuantityType<?>) command).doubleValue());
```

Comment:
This is not good as you could receive a value in Fahrenheit. You have to convert your quantity in Celsius first.
You can find example in other bindings (managing a target temperature).

Reply:
I return update channel in Celsius or the supplied unit?

Reply:
You are storing a decimal value and this value is considered in Celsius unit. So you have to convert the received QuantityType in unit Celsis.
Something like that:
```
                        QuantityType<Temperature> tempQtty = ((QuantityType<Temperature>) command)
                                .toUnit(SIUnits.CELSIUS);
                        if (tempQtty != null) {
                              car.setHvacTargetTemperature(tempQtty .doubleValue());
                       }
```

Reply:
That is nice I have used this idea.